src/components/dashboard/FaceRecognition/FRIntro.js

```diff
@@ -22,9 +22,8 @@ const FRIntro = props => {
   const { fullName } = store.get('profile')
   const { styles } = props
 
-  const isUnsupported = isIOS && isMobileSafari === false
+  const isUnsupported = !isMobileSafari
```

Comment:
should be isIOSWeb && isMobileSafari===false

Reply:
Changed because I thought it was kind of redundant. Is there any other mobile OS that runs Safari?

Reply:
probably not, just use ===